src/SlimSession/Helper.php

```diff
@@ -17,11 +17,11 @@ class Helper implements \ArrayAccess, \Countable, \IteratorAggregate
      * Get a session variable.
      *
      * @param string $key
-     * @param mixed  $default
+     * @param mixed $default
```

Comment:
Previous spacing is fine by me, leave as it was.

Reply:
reverted. thanks.